data_preparation/row_selection.py

```diff
@@ -64,7 +64,7 @@ def process(self, inputs):
             sys.exit(-1)
```

Comment:
By the way, could you change the error message in this `else` to be more explicative? Thank you

Reply:
Changed